quadrat/sass/blocks/_cover.scss

```diff
@@ -0,0 +1,36 @@
+.is-style-quadrat-cover-diamond{
+	&:after {
+		background-image: url(rotated.svg);
+		background-position: center;
+		background-repeat: no-repeat;
+		background-size: auto 70%;
+		content: "";
+		mix-blend-mode: soft-light;
+		position: absolute;
+		top: 0;
+		left: 0;
+		bottom: 0;
+		right: 0;
+		z-index: 0 !important; //this avoids weird interactions on the editor
```

Comment:
Do you know what line in the editor causes these issues?

Reply:
I think we don't need this anymore